cmd/livepeer/livepeer.go

```diff
@@ -88,8 +92,17 @@ func main() {
 		return
 	}
 
+	if *orchAddr != "" {
+		if *orchAddr == "" || *orchSecret == "" {
```

Comment:
hey! I'm not sure I understand why you're checking `if *orchAddr == ""` under the condition `if *orchAddr != ""` - should the second if always be false?

Reply:
Ah yep, fixed